apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/DeployNewReplicaPanel.tsx

```diff
@@ -366,23 +392,82 @@ const DeployNewReplicaPanel = ({
           </Listbox>
 
           <div className="flex flex-col gap-y-2">
-            <p className="text-foreground-light text-sm">
-              Read replicas will be on the same compute size as your primary database. Deploying a
-              read replica on the {selectedComputeMeta?.name} size incurs additional{' '}
-              <span className="text-foreground">{selectedComputeMeta?.price_description}</span>.
-            </p>
+            {showNewDiskManagementUI ? (
+              <>
+                <Collapsible_Shadcn_>
+                  <CollapsibleTrigger_Shadcn_ className="w-full flex items-center justify-between [&[data-state=open]>svg]:!-rotate-180">
+                    <p className="text-sm">
+                      Deploying a new replica will cost an additional $
+                      {estComputeMonthlyCost + additionalCostDiskSize}/month
+                    </p>
+                    <ChevronDown size={14} className="transition" />
+                  </CollapsibleTrigger_Shadcn_>
+                  <CollapsibleContent_Shadcn_ className="flex flex-col gap-y-1 mt-1">
+                    <p className="text-foreground-light text-sm">
+                      Read replicas will match the compute size of your primary database and will
+                      include 25% more disk space than the primary database to accommodate WAL
+                      files.
+                    </p>
+                    <p className="text-foreground-light text-sm">
+                      The additional cost per replica is based on your current compute size of{' '}
+                      <span className="text-foreground">{selectedComputeMeta?.name}</span> and your
+                      current disk size of{' '}
+                      <span className="text-foreground">
+                        {diskConfiguration?.attributes.size_gb}GB
+                      </span>
+                      , broken down as follows:
+                    </p>
+                    <Table>
+                      <TableHeader className="font-mono uppercase text-xs [&_th]:h-auto [&_th]:pb-2 [&_th]:pt-4">
+                        <TableRow>
+                          <TableHead className="w-[140px] pl-0">Item</TableHead>
+                          <TableHead>Description</TableHead>
+                          <TableHead className="text-right pr-0">Cost</TableHead>
+                        </TableRow>
+                      </TableHeader>
+                      <TableBody className="[&_td]:py-0 [&_tr]:h-[50px] [&_tr]:border-dotted">
+                        <TableRow>
+                          <TableCell className="pl-0">Compute size</TableCell>
+                          <TableCell>{selectedComputeMeta?.name}</TableCell>
+                          <TableCell className="text-right font-mono pr-0">
+                            ~${estComputeMonthlyCost}/month
```

Comment:
```suggestion
                            ${estComputeMonthlyCost}/month
```

We don't show the estimated sign for disk size cost either, so I'd opt to just skip it entirely